packages/@o3r/rules-engine/schemas/rulesets.schema.json

```diff
@@ -156,93 +156,21 @@
               "$ref": "#/definitions/RulesActionSetTemporaryFactBlock"
             },
             {
-              "$ref": "#/definitions/RulesActionUpdateConfigBlock"
+              "$ref": "https://raw.githubusercontent.com/AmadeusITGroup/otter/main/packages/%40o3r/configuration/schemas/rules-engine.config-action.schema.json"
```

Comment:
is it safe to always target the main branch?

Reply:
It is not but it will be complicated to target the tag (or branch) corresponding to the current version.
if you want we can create an task to address it